db/schema.sql

```
@@ -278,8 +284,13 @@ BEGIN
 END $$;
 
 -- New uniqueness covers (round, reporter, submission, claim-coalesced, client_nonce)
-CREATE UNIQUE INDEX IF NOT EXISTS ux_verification_verdicts_unique_nonce
-  ON verification_verdicts (round_id, reporter_id, submission_id, coalesce(claim_id, ''), (COALESCE(NULLIF(client_nonce, ''), '')));
+DROP INDEX IF EXISTS ux_verification_verdicts_unique_nonce;
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)